cms/templates/settings.html

```diff
@@ -199,6 +199,26 @@ <h3 class="title">${_("These Dates Are Not Used When Promoting Your Course")}</h
           </div>
           % endif
         </section>
+
+        <section class="group-settings details">
```

Comment:
I believe it will be ignored by the Drupal site. The reason I think it should be present in all cases is so that this information can migrate in time away from Drupal setting into here - it certainly will be redundant information, so if we want to hide and force a fix up of this field for all courses before cutover, we could hide it.